torchao/quantization/autoquant.py

```diff
@@ -422,6 +424,40 @@ def _quantized_linear_op(act_mat, w_qtensor, bias):
             y += bias
         return y
 
+
+class AQInt4G32WeightOnlyQuantizedLinearWeight(AffineQuantizedTensor, AQMixin):
```

Comment:
a bit of a mouthful? Also what does the mixin do exactly?

Reply:
it adds the autoquant functionality

Reply:
yeah but these are pretty insulated from users so it may be more valuable to be clear for devX